lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -240,6 +247,17 @@ spec = do
             res <- try' (runGet getAddress (BL.fromStrict $ unsafeFromHex hex))
             res `shouldSatisfy` isRight
 
+        describe "golden block0s from jcli" $ do
+            let dir = "test/data/block0s"
```

Comment:
I was lazy. To be consistent this should probably be `test/data/Cardano/Wallet/BinarySpec/block0s`.

The term "block0" vs "genesis-block" could also be debated.